src/index.js

```diff
@@ -0,0 +1,109 @@
+
+const increaseTemp = () => {
+    const tempElement = document.querySelector('#tempValue');
+    let currentTemp = Number(tempElement.innerHTML);
+    currentTemp++; 
+    tempElement.innerHTML = currentTemp;
+    setTempColor(currentTemp);
+    setLandscape(currentTemp);
+};
+
+const decreaseTemp = () => {
+    const tempElement = document.querySelector('#tempValue');
+    let currentTemp = Number(tempElement.innerHTML);
+    currentTemp--; 
+    tempElement.innerHTML = currentTemp;
+    setTempColor(currentTemp);
+    setLandscape(currentTemp)
+};
+
+const setLandscape = (currentTemp) => {
+    landscapeElement = document.querySelector('#landscape');
+    
+    if (currentTemp >= 80) {
+        landscapeElement.innerHTML = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    } else if (currentTemp >= 70) {
+        landscapeElement.innerHTML = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (currentTemp >= 60) {
+        landscapeElement.innerHTML = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else {
+        landscapeElement.innerHTML = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    }
+};
+
+const setTempColor = (currentTemp) => {
+    const tempElement = document.querySelector('#tempValue');
+    tempElement.classList.remove('red', 'orange', 'yellow', 'green', 'teal');
+
+    if (currentTemp >= 80) {
+        tempElement.classList.add('red');
+    } else if (currentTemp >= 70) {
+        tempElement.classList.add('orange');
+    } else if (currentTemp >= 60) {
+        tempElement.classList.add('yellow');
+    } else if (currentTemp >= 50) {
+        tempElement.classList.add('green');
+    } else {
+        tempElement.classList.add('teal');
+    }
+};
+
+const cityInput = (event) => {
+    const cityHeader = document.querySelector('#headerCityName');
+
+    cityHeader.innerHTML = event.target.value;
+};
+
+const setTempFromCity = async () => {
+    const cityElement = document.querySelector('#headerCityName');
+    const cityName = cityElement.innerHTML;
+
+    const cityLocationInfo = await axios.get(`${API_KEY}/location?q=${cityName}`);
+
+    const tempElement = document.querySelector('#tempValue');
+    const cityLat = cityLocationInfo.data[0].lat;
+    const cityLon = cityLocationInfo.data[0].lon;
+
+    const cityTempInfo = await axios.get(`${API_KEY}/weather?lat=${cityLat}&lon=${cityLon}`)
+    
+    const cityTempKelvin = cityTempInfo.data.main.temp;
+    const cityTempFaren = Math.round((cityTempKelvin * 1.8) - 459.67);
+
+    tempElement.innerHTML = cityTempFaren;
+
+    setTempColor(cityTempFaren);
+    setLandscape(cityTempFaren);
+};
+
+const setSky = (event) => {
+    const skyElement = document.querySelector('#sky');
+    const skyOptions = ["☁️ ☁️ ☁️ ☀️ ☁️ ☁️", "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️", "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧", "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"]
+    skyElement.innerHTML = skyOptions[event.target.value]
+}
+
+const resetCity = () => {
+    const cityField = document.querySelector('#cityNameInput');
+    cityField.value = '';
+
+    const cityElement = document.querySelector('#headerCityName');
```

Comment:
`document.querySelector('#headerCityName');` appears in your code 3 times. While it's not a ton of repetition, it still is three function calls and these calls aren't complex, time intensive operations. 

However, each time we type it, we could introduce a bug. 

I'd prefer a global variable that can be referenced throughout the file or create an object with all the elements you'll need, like we did in our livecode: https://github.com/Ada-Activities/Pick-Me-Up-Pup/blob/c23-solution/src/index.js#L4-L10